packages/stackgen/package.json

```diff
@@ -0,0 +1,339 @@
+{
+  "name": "stackgen",
+  "license": "Apache-2.0",
+  "version": "0.0.1",
+  "author": {
+    "name": "Justin McCormick",
+    "email": "me@justinmccormick.com"
+  },
```

Comment:
shall we refactor this? 😁

Reply:
yeah let's do it in another PR. We need to setup a name / email address.